platformio.ini

```diff
@@ -12,3 +12,6 @@
 platform = teensy
 board = teensy41
 framework = arduino
+upload_protocol = teensy-cli
+lib_deps =
+    FlexCAN_T4
```

Comment:
This is what was breaking Clion, you had included the systems version of flexcan from another project, but never actually added it to compilation

Reply:
Lol that'll do it